src/modules/assertions/require_max_file_number.ts

```diff
@@ -0,0 +1,13 @@
+import { requireEvent } from "#/assertions";
+import { PR } from "src/domain";
+import { github } from "src/infra/github";
+
+export const  requireMaxFileNumber = async (pr: PR) => {
+  const max_files_allowed = 100;
+  requireEvent();
```

Comment:
```suggestion
```
I think this is unnecessary?  You don't use the return value anywhere, and it already is being executed in `_main_` before this.

Reply:
This was marked as resolved, but I don't see a comment or a fix for it?